docs/examples/milestones/af-implementation-milestones-1.json

```diff
@@ -0,0 +1,125 @@
+{
+  "uri": "https://standard.open-contracting.org/examples/release/ocds-afalqw-001-5c10b8c57e0a947f6b212f58-001",
+  "publisher": {
+    "name": "National Procurement Authority, Afghanistan"
+  },
+  "publishedDate": "2018-09-21T08:00:00+04:30",
+  "version": "1.1",
+  "releases": [
+    {
+      "language": "prs-AF",
+      "contracts": [
+        {
+          "status": "active",
+          "title": "School repairs and mantainance projects, Rahman Baba & Khushal Khan Khattak",
+          "implementation": {
+            "milestones": [
+              {
+                "status": "notMet",
+                "title": "Finish the exterior and interior walls",
+                "dueDate": "2018-10-10T00:00:00+04:30",
+                "type": "financing",
+                "id": "20653Financing"
+              },
+              {
+                "status": "notMet",
+                "title": "Finish the exterior and interior walls",
+                "dueDate": "2018-10-05T00:00:00+04:30",
+                "type": "delivery",
+                "id": "20653Delivery"
+              },
+              {
+                "status": "notMet",
+                "title": "Repair of the electric system",
+                "dueDate": "2018-10-31T00:00:00+04:30",
+                "type": "financing",
+                "id": "20654Financing"
+              },
+              {
+                "status": "notMet",
+                "title": "Repair of the electric system",
+                "dueDate": "2018-10-28T00:00:00+04:30",
+                "type": "delivery",
+                "id": "20654Delivery"
```

Comment:
Is this a real example? I'm a bit confused why the milestone titles are the same for the delivery and financing milestones.

I would expect that the financing milestones would have titles like 'Payment for repair of electric system' etc.

Reply:
@duncandewhurst  Yes, this is a real example indeed https://ocds.ageops.net/api/release/5c10b8c57e0a947f6b212f58
(with the titles translated) should I change them?

Reply:
Yes, I think it would be good to update them to make the example clearer.